### DISCUSSION TITLE
Using the Dark Mode with Global Style in the providers does not lock down the application to one specific mode

### DISCUSSION THREAD

**Discussion by techwithtwin** [chakra-ui/chakra-ui#9106] [General] [answered]
### Description

When I wrapped my entire project with DarkMode and add a GlobalStyle component the page was not locked into dark mode.
There is no reproduction for this. 

![image](https://github.com/user-attachments/assets/3f45884e-a47f-4a24-8b71-028aab1f2301)

### Link to Reproduction

https://stackblitz.com/edit/chakra-ui-v3?file=src%2FApp.tsx

### Steps to reproduce

2. Click on '...'
3. Scroll down to '...'

### Chakra UI Version

2.10.1

### Browser

Google Chrome

### Operating System

- [ ] macOS
- [ ] Windows
- [X] Linux

**Comment by segunadebayo** [comment, marked as answer]
`DarkMode` plus `GlobalStyle` is v2. That pair does not lock color mode in a reliable way.

In v3 use `ColorModeProvider forcedTheme="dark"`. https://chakra-ui.com/docs/components/concepts/color-mode
